haskell/src/Monocle/Env.hs

```diff
@@ -41,13 +41,17 @@ instance MonadFail AppM where
   fail = error . toText
 
 -- | 'getConfig' reload the config automatically from the env
-getConfig :: AppM [Config.Index]
+getConfig :: AppM Config.Config
 getConfig = do
   loadConfig <- asks config
   --  logger <- asks (glLogger . aEnv)
   --  let logReload = (logEvent logger . ReloadConfig)
   liftIO (snd <$> loadConfig)
 
+-- | 'getWorkspaces' reload the workspaces automatically from the env
+getWorkspaces :: AppM [Config.Index]
```

Comment:
nit: Config.getWorkspaces does something different. It would be nice to use different name too, for example this one could be `askWorkspaces`, or the other one could be a `from` provided by `instance From Config Workspace`